_topic_map.yml

```diff
@@ -2156,64 +2156,62 @@ Name: Service Mesh
-#     File: ossm-traffic-manage
-#   - Name: Custom resources
-#     File: ossm-custom-resources
-#   - Name: Extensions
-#     File: ossm-extensions
-#   - Name: Using the 3scale Istio adapter
-#     File: threescale-adapter
-- Name: Service Mesh 1.x
-  Dir: v1x
-  Topics:
-  - Name: Service Mesh 1.x release notes
-    File: servicemesh-release-notes
-  - Name: Service Mesh architecture
-    File: ossm-architecture
-  - Name: Service Mesh and Istio differences
-    File: ossm-vs-community
-  - Name: Preparing to install Service Mesh
-    File: preparing-ossm-installation
-  - Name: Installing Service Mesh
-    File: installing-ossm
-  - Name: Customizing the installation
-    File: customizing-installation-ossm
-  - Name: Deploying applications on Service Mesh
-    File: prepare-to-deploy-applications-ossm
-  - Name: Data visualization and observability
-    File: ossm-observability
-  - Name: Security
-    File: ossm-security
-  - Name: Traffic management
-    File: ossm-traffic-manage
-  - Name: Custom resources
-    File: ossm-custom-resources
-  - Name: Using the 3scale Istio adapter
-    File: threescale-adapter
+ - Name: Service Mesh 2.x
+   Dir: v2x
+   Topics:
+   - Name: Service Mesh 2.x release notes
+     File: servicemesh-release-notes
+   - Name: Service Mesh architecture
+     File: ossm-architecture
+   - Name: Service Mesh and Istio differences
+     File: ossm-vs-community
+   - Name: Preparing to install Service Mesh
+     File: preparing-ossm-installation
+   - Name: Installing Service Mesh
+     File: installing-ossm
+   - Name: Upgrading from 1.1 to 2.0
+     File: upgrading-ossm
+   - Name: Customizing the installation
+     File: customizing-installation-ossm
+   - Name: Deploying applications on Service Mesh
+     File: prepare-to-deploy-applications-ossm
+   - Name: Data visualization and observability
+     File: ossm-observability
+   - Name: Security
+     File: ossm-security
+   - Name: Traffic management
+     File: ossm-traffic-manage
+   - Name: Custom resources
+     File: ossm-custom-resources
+   - Name: Extensions
+     File: ossm-extensions
+   - Name: Using the 3scale Istio adapter
+     File: threescale-adapter
+ - Name: Service Mesh 1.x
+   Dir: v1x
+   Topics:
+   - Name: Service Mesh 1.x release notes
+     File: servicemesh-release-notes
+   - Name: Service Mesh architecture
+     File: ossm-architecture
+   - Name: Service Mesh and Istio differences
+     File: ossm-vs-community
+   - Name: Preparing to install Service Mesh
+     File: preparing-ossm-installation
+   - Name: Installing Service Mesh
+     File: installing-ossm
+   - Name: Customizing the installation
+     File: customizing-installation-ossm
+   - Name: Deploying applications on Service Mesh
+     File: prepare-to-deploy-applications-ossm
+   - Name: Data visualization and observability
+     File: ossm-observability
+   - Name: Security
+     File: ossm-security
+   - Name: Traffic management
+     File: ossm-traffic-manage
```

Comment:
It looks like Custom resources in 1x got removed.

Reply:
Updated